ros2_components_ws/src/cb_hw_fw_pos_read_pos_vel/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
This results in a warning in CMake 3.31 . I would either raise the `policy_max` or directly the cmake minimum version.

Reply:
I changed it (and also the one for the other packages in the ros2_components_ws folder) to match the one of the repo (which is, right now, 3.17)